applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+'require form';
+'require fs';
+'require view';
+
+return view.extend({
+	load: function() {
+		return fs.list('/dev').then(function(devs) {
+			return devs.filter(function(dev) {
+				return dev.name.match(/^[sh]d[a-z]$/);
```

Comment:
Maybe filter for `dev.type == "file"` too, just to be on the safe side.

Reply:
Seems the hdd devices should filter for `dev.type == 'block'`?